app/user/rest.py

```diff
@@ -84,6 +84,7 @@ def create_user():
 
     save_model_user(user_to_create, password=req_json.get("password"), validated_email_access=True)
     result = user_to_create.serialize()
+    log_user(result, "User created")
```

Comment:
Just checking, what's included in this log message, it's not easy to see by logging at the code what `user_to_create.serialize()` would log.

Reply:
this will log out the User object, giving something like this:
`{
    "message": "User created",
    "id": "ab073fd6-5c8a-4c79-a4cc-e0aa281f3095", 
    "email_address": "test.user2@digital.cabinet-office.gov.uk", 
    "user_id": "eeb7c729-31f1-4c15-97e0-4678f936fd4f", 
    "service_id": "notify-admin", 
    "created_at": "2024-06-19T10:18:00.550144Z", 
    "status": "pending", 
    "auth_type": "sms_auth", 
   ...
}`